app/Models/Items/Item.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Models\Items;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Item extends Model
+{
+    protected $table = 'items';
+
+    public function type()
+    {
+        return $this->belongsTo(ItemType::class);
+    }
+
+    public function subtype()
+    {
+        return $this->belongsTo(ItemType::class);
+    }
+
+    public function modifiers() {
+        return $this->hasMany(
+            Modifier::class,
+            'object_id'
+        )
+            ->where('object_class', self::class);
+    }
```

Comment:
Yees

Reply:
Polymorphic relation though `->morphMany()` `->morphTo.()`

Reply:
I've don't this, had to rename object_class column to object_type as that is what laravel defaults to. Also needed to install doctrine as I couldn't run migrations until this..